benchmarks/mruby-2018-05-23/Dockerfile

```diff
@@ -17,7 +17,10 @@
 FROM gcr.io/oss-fuzz-base/base-builder@sha256:fb1a9a49752c9e504687448d1f1a048ec1e062e2e40f7e8a23e86b63ff3dad7c
 RUN apt-get update && apt-get install -y build-essential ruby bison ninja-build \
     cmake zlib1g-dev libbz2-dev liblzma-dev
-RUN git clone --depth 1 https://github.com/mruby/mruby mruby
+RUN git clone \
```

Comment:
CC @andreafioraldi I assume if we upgrade this benchmark we will lose the bugs it was finding?

Reply:
yes ofc this is a vulnerable version